book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
```

Comment:
- [ ] Getters and Setters (attr_accessor) for @rentals should be present, please fix that.